tools/configure_for_migration_analytics/tasks/copy-sample-profile-to-default.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: CFME | Set the default SmartState profile | Check if the default profile exists
+  shell: |
+    set -o pipefail                   \
+    source {{ miq_profile }} &&       \
+    {{ miq_home_dir }}/bin/rails r "  \
+      default_si = ScanItemSet.find_by(:name => 'default')
```

Comment:
Why are we modifying the default scanitemset? Is this for upgraded customers or something?

Reply:
we're copying the 'sample' scanitemset to 'default', but only if 'default' doesn't already exist